telemetry/specs/INT.mdk

```diff
@@ -1898,14 +1901,128 @@ INT-MX Header, followed by UDP payload:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-## Example with INT-MD over IPv4/GRE
+## Example with INT-MD over IPv4/GRE (Original packet IPv4)
+
+In this scenario host1 sends an IPv4 packet to host2.
+
```

Comment:
In the other examples the first sentence and the next part are in the same paragraph, so this blank line should be removed.

Reply:
Done